README.md

```diff
@@ -28,6 +28,14 @@ This action expects variables for:
 - `DEST`, the name of the destination directory you would like the site to build to,
 - `TOKEN`, in the form `${{ secrets.TOKEN }}` assuming `TOKEN` is the name you've used.
 
+You can optionally set a string value for the variable named `HUGO_ARGS`. Note these should be set on a single line.
+
+- `HUGO_ARGS` in the form 'string' Here are some examples:
+- `HUGO_ARGS: '--minify'`
+- `HUGO_ARGS: '-D -F'`
+  
+If you do not wish to pass any arguments. Please leave HUGO_ARGS blank as `HUGO_ARGS:`
```

Comment:
Now that the script does `${HUGO_ARGS:-""}`, the user shouldn't need to set `HUGO_ARGS` as blank, they should just be able to omit the whole `env:` thing

Reply:
First, Let me apologise for the rubbish english I have been writing lately. I have not been paying attention to the auto correct.
Second, You are absolutely correct. This is a good example of defensive programming and makes for a far more robust implementation.

Reply:
Merged in. Thank you both!